app/javascript/utils/form/final_form/MultiSelectField.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+import { Field } from 'react-final-form'
+
+import MultiSelect from 'components/molecules/MultiSelect'
+
+import { FieldError, HelpText, Label } from './Field'
+
+const format = (options) => {
+  return options && options.map((option) => (typeof option === 'string' ? option : option.value))
+}
+const HEIGHT_NORMAL = 'normal'
+const HEIGHT_SMALL = 'small'
+
+const MultiSelectField = ({
+  blockNote,
+  disabled = false,
+  fieldName,
+  height = HEIGHT_NORMAL,
```

Comment:
We;re not going to support HEIGHT_SMALL anymore, right? If so I would remove this prop (you can leave it in MultiSelect but just always hardcode the height to "normal" from MultiSelectField.